.github/workflows/ci.yml

```diff
@@ -42,15 +49,9 @@ jobs:
 
       - name: Test with pytest
         run: |
-          pip install pytest
+          pip install pytest "numpy${{ matrix.numpy-version }}"
```

Comment:
Why not in the install step?

Reply:
Because it ist part of the test setup. The install/build step is separate. That one actually tests the installability from the runtime requirements,

https://github.com/pydata/bottleneck/blob/787d6daa292ef013efb2ce93f100079457330363/setup.py#L195

which installs numpy 1.26 at the moment.

https://github.com/pydata/bottleneck/actions/runs/9022389505/job/24849379279?pr=450#step:4:31